webClient/src/environments/environment.prod.ts

```diff
@@ -14,6 +14,20 @@ export const environment = {
   production: true
 };
 
+export const prod_config = {
```

Comment:
Generally, we don't declare globals like this, check out this example from Editor

`import { KeyCode } from '../../shared/keycode-enum';`

https://github.com/zowe/zlux-editor/blob/staging/webClient/src/app/shared/keycode-enum.ts